tests/test_fillnodata.py

```diff
@@ -42,20 +38,8 @@ def test_fillnodata_invalid_types():
         fillnodata(a, 42)
 
 
-def test_fillnodata_mask_ones():
-    # when mask is all ones, image should be unmodified
-    a = np.ones([3, 3]) * 42
-    a[1][1] = 0
-    mask = np.ones([3, 3])
-    result = fillnodata(a, mask)
-    assert(np.all(a == result))
-
-'''
-def test_fillnodata_smooth():
-    a = np.array([[1,3,3,1],[2,0,0,2],[2,0,0,2],[1,3,3,1]], dtype=np.float64)
-    mask = ~(a == 0)
-    result = fillnodata(a, mask, max_search_distance=1, smoothing_iterations=0)
-    assert(result[1][1] == 3)
-    result = fillnodata(a, mask, max_search_distance=1, smoothing_iterations=1)
-    assert(round(result[1][1], 1) == 2.2)
-'''
+def test_fillnodata_mask_ones(hole_in_ones):
+    """when mask is all ones, image should be unmodified"""
+    mask = np.ones((5, 5))
+    result = fillnodata(hole_in_ones, mask)
+    assert(np.all(hole_in_ones == result))
```

Comment:
Why not match your above pattern, for consistency?

`assert (hole_in_ones == result).all()`